### ISSUE TITLE
Unable to use cbp-theme.css from dist

### ISSUE THREAD

**Issue by jscharett** [US-CBP/cbp-theme#241]
When trying to update to the latest 1.9.0 from 1.8.1, the main css no longer links to the font files correctly.

I get multiple errors like the folllowing
```
WARNING in ./node_modules/raw-loader!./node_modules/postcss-loader/lib??embedded!./node_modules/cbp-theme/dist/cbp-theme.css
(Emitted value instead of an instance of Error) postcss-url: C:\Users\ZGCFXPC\workspace\atsp-udr2-ui-main\node_modules\cbp-theme\dist\cbp-theme.css:10497:3: Can't read file 'C:\Users\ZGCFXPC\workspace\atsp-udr2-ui-main\node_modules\cbp-theme\dist\roboto-fontface\fonts\roboto\Roboto-BlackItalic.eot', ignoring
```

**Comment by jscharett** [comment]
I wonder if #91 may have something to do with it?  It also looks like the cbp-theme.css in dist is no longer being compressed?  In 1.8.1, it was compressed, but in 1.9.0, it is not.  Maybe there is a compilation step missing from the build?

**Comment by davezen1** [comment]
This seems to be a build issue. For a temp work around you can use the cbp-theme.min.css which has the proper links to the font files.

**Comment by jscharett** [comment]
ah, so starting with 1.9.0, there will be both a minified and non minified version of the theme css?  Previously, there was just cbp-theme.css which was minified.  will give it a shot, thanks.

**Comment by jscharett** [comment]
Actually, there does not appear to be a cbp-theme.min.css in the package.  cbp-theme.css is there in dist, along with cbp-theme.js and cbp-theme.min.js, but not minified css.  Am i missing something?

**Comment by davezen1** [comment]
It should be there along with a map

cbp-theme.min.css
cbp-theme.min.css.map
cbp-theme.css

could be your IDE , check from command line.

**Comment by jscharett** [comment]
You are correct.  Must have something to do with the .map files

**Comment by davezen1** [comment]
Fixed in v1.10.2
